ckan/config/middleware/__init__.py

```diff
@@ -33,6 +33,9 @@ def make_app(conf: Union[Config, CKANConfig]) -> CKANApp:
     # Set this internal test request context with the configured environment so
     # it can be used when calling url_for from tests
     global _internal_test_request_context
-    _internal_test_request_context = flask_app._wsgi_app.test_request_context()
+    _internal_test_request_context = getattr(
+        flask_app,
+        "_wsgi_app",
+    ).test_request_context()
```

Comment:
What's getattr here for? isn't this just a slower version of `flask_app._wsgi_app.test_request_context()`?

Reply:
The same as below, I did it to avoid complaints about undefined attribute - reverting it back, as it isn't any safer than direct access to the attribute